platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -170,6 +178,12 @@ public void setAnnouncements(List<String> announcements) {
         sortAnnouncementsByDate();
     }
 
+    public void setLinks(List<String> links) {
+        this.links = links.stream()
+                .filter(link -> !link.isEmpty())
+                .collect(Collectors.toCollection(LinkedList::new));
```

Comment:
I don't care about random access, just insertions / deletions (cause I want to ignore blank links), which a linkedlist should theoretically be better for. I can change it to whatever though. There are so few elements I doubt it matters.